mteb/tasks/Classification/multilingual/AfriSentiClassification.py

```diff
@@ -0,0 +1,53 @@
+from __future__ import annotations
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks import AbsTaskClassification, MultilingualTask
```

Comment:
replace it with `from mteb.abstasks import AbsTaskClassification, MultilingualTask`